standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/metrics/MetricsConstants.java

```diff
@@ -23,7 +23,9 @@
   public static final String COMPACTION_STATUS_PREFIX = "compaction_num_";
   public static final String COMPACTION_OLDEST_ENQUEUE_AGE = "compaction_oldest_enqueue_age_in_sec";
   public static final String COMPACTION_INITIATOR_CYCLE = "compaction_initiator_cycle";
+  public static final String COMPACTION_FAILED_INITIATOR_CYCLE = "compaction_failed_initiator_cycle";
```

Comment:
I would rename both metrics to COMPACTION_FAILED_INITIATOR_RATIO(compaction_failed_initiator_ratio), COMPACTION_FAILED_CLEANER_RATIO(compaction_failed_cleaner_ratio)

Reply:
Renamed.